iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -2528,7 +2575,7 @@ static PMQTTTRANSPORT_HANDLE_DATA InitializeTransportHandleData(const IOTHUB_CLI
                 }
                 else
                 {
-                    /* Codes_SRS_IOTHUB_MQTT_TRANSPORT_07_008: [If the upperConfig contains a valid protocolGatewayHostName value the this shall be used for the hostname, otherwise the hostname shall be constructed using the iothubname and iothubSuffix.] */
+                    /* Codes_SRS_IOTHUB_MQTT_TRANSPORT_07_008: [If the upperConfig contains a valid protocolGatewayHostName value this shall be used for the hostname, otherwise the hostname shall be constructed using the iothubname and iothubSuffix.] */
```

Comment:
Yeah, you had to bring that up didn't you:)?

Anyway I had same idea and it turns out that this requirement is not referenced anywhere else.  There is one with a matching code, but it's a different value.  (That one is at least consistent throughout.)

```
IoTHubTransportMqtt_GetSendStatus shall get the send status by calling into the IoTHubMqttAbstract_GetSendStatus function
```

Since we're not doing these reqs for anything than legacy, I didn't feel inclined to add a new req for this one off.  I'd almost rather delete this one than do that.